docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,59 @@ parent = "smn_cli"
 
     Creates a new network with a name specified by the user
 
-      -d, --driver=      Driver to manage the Network
-      --help=false       Print usage
+    --aux-address=map[]      Auxiliary ipv4 or ipv6 addresses used by network driver
+    -d, --driver=DRIVER      Driver to manage the Network. Bridge is the default.
+    --gateway=[]             ipv4 or ipv6 Gateway for the master subnet
+    --help=false             Print usage
+    --ip-range=[]            Allocate container ip from a sub-range
+    --ipam-driver=default    IP Address Management Driver
+    --subnet=[]              Subnet in CIDR format that represents a network segment
+    --help=false             Print usage
 
-Creates a new network that containers can connect to. If the driver supports multi-host networking, the created network will be made available across all the hosts in the cluster. Daemon will do its best to identify network name conflicts. But its the users responsibility to make sure network name is unique across the cluster. You create a network and then configure the container to use it, for example:
+Creates a new network. Driver `DRIVER` accepts `bridge` or `overlay` which are the built-in network drivers. If you have installed a 3rd party or your own custom network driver, you can supply the `DRIVER` provided also. If you don't specify the `--driver` option, Docker automatically creates a `bridge` network for you.
+
+When you install Docker Engine, creates a `bridge` network automatically. This network corresponds to the `docker0` bridge that Docker has always relied on. When you launch a new container, it automatically launches on this bridge network. You cannot remove this default bridge network but you can create new ones using the `network create` command.
+
+```
+$ docker network create -d bridge my-bridge-network
+```
+
+Bridge networks support a isolated networks on the Docker host machine. If you want to create a network that spans multiple Docker hosts, you must create an `overlay` network.
```

Comment:
@dave-tucker good question...idk. The rest of the documentation refers to "Docker host" more often than Docker Engines. "Docker Engines" is a new way to approach... and I'm not against it. We just need to establish it more as a concept.